tests/NSubstitute.Analyzers.Tests.CSharp/CodeFixProviderTests/NonVirtualSetupAnalyzerSuppressDiagnosticsCodeFixProviderTests/NonVirtualSetupSuppressDiagnosticsCodeFixActionsTests.cs

```diff
@@ -33,9 +33,9 @@ public void Test()
 }";
             await VerifyCodeActions(source, new[]
             {
-                "Suppress NS001 for indexer this[] in nsubstitute.json",
-                "Suppress NS001 for class Foo in nsubstitute.json",
-                "Suppress NS001 for namespace MyNamespace in nsubstitute.json"
+                "Suppress NS1000 for indexer this[] in nsubstitute.json",
```

Comment:
Maybe use string interpolation here (and below) with `{DiagnosticIdentifiers. NonVirtualSetupSpecification}` to make sure they stay in sync?

Not sure if that is counterproductive for tests. 🤔

Reply:
Done